io.github.wxmaxima_developers.wxMaxima.yml

```diff
@@ -0,0 +1,64 @@
+app-id: io.github.wxmaxima_developers.wxMaxima
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+
+finish-args:
+      - --share=ipc
+      - --socket=x11
+      - --env=GNUPLOT_DRIVER_DIR=/app/libexec/gnuplot/5.4/
+      - --filesystem=host:rw
+      
+modules:
+  - name: gnuplot
+    buildsystem: autotools
+    no-autogen: true
+    config-opts:
+      - --with-qt=no
+    sources:
+      - type: archive
+        url: https://prdownloads.sourceforge.net/gnuplot/5.4.10/gnuplot-5.4.10.tar.gz
+        sha256: 975d8c1cc2c41c7cedc4e323aff035d977feb9a97f0296dd2a8a66d197a5b27c
+  - name: sbcl
+    buildsystem: simple
+    no-autogen: true
+    build-commands:
+      - cp -r ./sbcl/* /app/
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/f/sbcl-2.3.9.tar.gz
```

Comment:
what about using the official download too?
http://www.sbcl.org/platform-table.html

Reply:
And build from source, because:
```
/bin/sh: line 1: /app/bin/sbcl: cannot execute binary file: Exec format error
```

(on aarch64)

Reply:
To compile sbcl you need sbcl. I'm looking for an alternative

Reply:
from the page linked above:

![image](https://github.com/flathub/flathub/assets/114441/bfda7408-8d34-4fdb-aab7-00d682b01e23)